web/components/nav/Navbar.tsx

```diff
@@ -0,0 +1,44 @@
+import { createClient } from '@/utils/supabase/server';
+import { getUserAvatarUrl } from '@utils/lib/common/user_helper';
+import { cookies } from 'next/headers';
+import Image from 'next/image';
+import Link from 'next/link';
+import { StyledLink } from '../common/link';
+import { NotificationPin } from '../notification/NotificationPin';
+import { NavLink } from './NavLink';
+
+export default async function Navbar() {
+  const cookieStore = cookies();
+  const supabase = createClient(cookieStore);
+
+  const {
+    data: { user },
+  } = await supabase.auth.getUser();
+
+  const userAvatarUrl = user ? getUserAvatarUrl(user) : null;
+
+  return (
+    <nav className='flex h-16 w-full max-w-5xl items-center justify-between gap-4 p-3 text-sm'>
```

Comment:
Comme ça ça évite le sursaut qu'on a à cause des différentes hauteurs de contenu.
(Et tu peux voir ce que donne les animate sur la vidéo)

https://github.com/victorbillaud/tomato/assets/91191933/6ee1882c-6dc3-4bd3-bfa5-ca89b410716b

```suggestion
    <nav className='flex w-full max-w-5xl items-center justify-between gap-4 p-3 text-sm'>
```